icen_PUIR/common/pop_types.txt

```diff
@@ -308,16 +308,16 @@ migration_chance =
 		}
 	}
 
-	modifier = {
-		factor = -1.0
-		is_accepted_culture = no
-		NOT = {
-			OR = {  
-				culture = irish
-				state_scope = { is_colonial = yes }
-			} 
-		}
-	}
+	#modifier = {

```

Comment:
It's completely okay to delete code instead of just commenting it out, you know. :)